Sources/Authenticator/Models/Internal/AuthenticatorMFAType.swift

```diff
@@ -5,7 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-enum MFAType {
+enum AuthenticatorMFAType {
```

Comment:
nit: I liked the one without the prefix better, especially with it being an internal type (a la `Step`)

Reply:
`MFAType` in Authenticator and Amplify doesn't resolve correctly because of the naming collisions.. Thats why I renamed